x/payment/spec/docs.md

```diff
@@ -158,13 +158,6 @@ if err != nil {
 
 ### How the commitments are generated
 
-1. create the final version of the message by adding the length delimiter, the namespace, and then the message together into a single string of bytes
-
-   ```python
-   finalMessage = [length delimiter] + [namespace] + [message]
-   ```
-
-2. chunk the finalMessage into shares of size `appconsts.ShareSize`
-3. pad until number of shares is a power of two
-4. create the commitment by aranging the shares into a merkle mountain range
-5. create a merkle root of the subtree roots
+1. Split the message into shares of `appconsts.ShareSize`
+1. Arrange the shares into a Merkle mountain range where each tree in the mountain range has a maximum size of `squareSize`
+1. Take the roots of the trees in the Merkle mountain range and create a new Merkle tree. The message share commitment is the Merkle root of this Merkle tree.
```

Comment:
```suggestion
2. Arrange the shares into a Merkle mountain range where each tree in the mountain range has a maximum size of `squareSize`
3. Take the roots of the trees in the Merkle mountain range and create a new Merkle tree. The message share commitment is the Merkle root of this Merkle tree.
```

Reply:
FWIW in Markdown it is possible to create an ordered list by prefixing each item with `1.`. This approach can be easier to maintain because one doesn't need to manually modify numbers when re-arranging items in the list. I'm not strongly opinionated on either style but just wanted to share what it looks like in rendered markdown with `1.` prefixes:

## Screenshot

<img width="965" alt="Screen Shot 2022-10-26 at 3 27 06 PM" src="https://user-images.githubusercontent.com/3699047/198038196-35674b0d-0ad2-48a1-b6ed-662a733b4967.png">

Reply:
Aaah, didn't know about this. I'm fine with any